### DISCUSSION TITLE
"Overwhelmed by notifications?"

### DISCUSSION THREAD

**Discussion by ganego** [community/community#58] [Other Feature Feedback, Questions, & Ideas] [answered]
So you ask me "Overwhelmed by notifications?".

Yes.
Want to know why?
Because someone thought it would be a genius idea to put every notification for every repo in one giant list without any sorting.  

You know what would solve this?  
Separating notifications by repo.  
Just like it was for years before someone "enhanced" the design. 
And no, I do not want to click on each single repo to apply a filter first for something that was easily done before without any user interaction.

**Comment by BenJam** [comment, marked as answer]
:wave: hey @ganego we heard this loud and clear when we first announced that we were updating the notifications experience on GitHub.com — and we implemented a 'group by repo' option to do just that:

![Screenshot 2020-11-10 at 17 46 08](https://user-images.githubusercontent.com/158833/98711257-ae2c7c80-237c-11eb-98c6-d0183f7701ae.png)

We've got some updates that [we've already started rolling out](https://github.blog/changelog/2020-11-10-unwatch-recommendations/) that should help you reduce the noise, so keep an eye on the changelog and the blog for those next week.

**Reply by ganego** [reply]
Has the "Group by repo" been there all the time since the new system was implemented some months ago and I just overlooked it all the time?  
Well it works, thank you.

**Reply by BenJam** [reply]
It has been there since we replaced the previous experience with new notifications, but we have evidently not done a good enough job of making people aware of it, so we'll address that 👍
